src/bosh_azure_cpi/lib/cloud/azure/vms/vmss_manager.rb

```diff
@@ -29,61 +29,60 @@ def create(bosh_vm_meta, vm_props, network_configurator, env)
       vmss_name = _get_vmss_name(vm_props, env)
       _ensure_resource_group_exists(resource_group_name, vm_props.location)
 
-      existing_vmss = @azure_client.get_vmss_by_name(resource_group_name, vmss_name)
-      vmss_params = {}
+      vmss_params = {
+        resource_group_name: resource_group_name
+      }
```

Comment:
It seems that `vmss_params` are re-defined in line 37.

Reply:
resolved.